packages/clay-tabs/src/__tests__/index.tsx

```diff
@@ -0,0 +1,143 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import '@testing-library/jest-dom/extend-expect';
+import ClayTabs from '..';
+import React from 'react';
+import {cleanup, fireEvent, render} from '@testing-library/react';
+
+const ClayTabsWithItems = () => {
+	const [activeTabKeyValue, setActiveTabKeyValue] = React.useState<number>(0);
+
+	return (
+		<>
+			<ClayTabs>
+				<ClayTabs.Item
+					active={activeTabKeyValue == 0}
+					onClick={() => setActiveTabKeyValue(0)}
+				>
+					{'Dummy1'}
+				</ClayTabs.Item>
+				<ClayTabs.Item
+					active={activeTabKeyValue == 1}
+					data-testid="tabItem2"
+					onClick={() => setActiveTabKeyValue(1)}
+				>
+					{'Dummy2'}
+				</ClayTabs.Item>
+				<ClayTabs.Item
+					active={activeTabKeyValue == 2}
+					onClick={() => setActiveTabKeyValue(2)}
+				>
+					{'Dummy3'}
+				</ClayTabs.Item>
+			</ClayTabs>
+			<ClayTabs.Content activeTabKey={activeTabKeyValue}>
+				<ClayTabs.TabPane data-testid="tabPane1">
+					{'Tab Content 1'}
+				</ClayTabs.TabPane>
+				<ClayTabs.TabPane data-testid="tabPane2">
+					{'Tab Content 2'}
+				</ClayTabs.TabPane>
+				<ClayTabs.TabPane>{'Tab Content 3'}</ClayTabs.TabPane>
+			</ClayTabs.Content>
+		</>
+	);
+};
+
+describe('ClayTabs', () => {
+	afterEach(cleanup);
+
+	it('renders', () => {
+		const {container} = render(<ClayTabs />);
+
+		expect(container).toMatchSnapshot();
+	});
+
+	it('renders justifying nav items', () => {
+		const {container} = render(<ClayTabs justified />);
+
+		expect(container).toMatchSnapshot();
```

Comment:
Instead of adding a bunch of snapshot tests, what do you think about testing explicit dom items? For example, in this case we could check `expect(container.querySelector('.nav.nav-justified')).toBeTruthy()`. This would at least help eliminate an accidental or blind update to all snapshots. And this way we also target specific truthy statements rather than the whole dom.

Reply:
Oh yeah, I agree. I will take care of updating these tests to eliminate some of the snapshots.